python/ribasim/ribasim/model.py

```diff
@@ -738,3 +741,67 @@ def _add_allocation(self, uds):
             uds[varname] = da
 
         return uds
+
+    def to_fews(
+        self,
+        region_home: str | PathLike[str],
+        network: bool = True,
+        results: bool = True,
+    ) -> None:
+        """
+        Write the model network and results into files used by Delft-FEWS.
+
+        ** Warning: This method is experimental and is likely to change. **
+
+        To run this method, the model needs to be written to disk, and have results.
+        The Node and Link tables are written to shapefiles in the Config directory.
+        The results are written to NetCDF files in the Modules directory.
+        The netCDF files are NetCDF4 with CF-conventions.
+
+        Parameters
+        ----------
+        region_home: str | PathLike[str]
+            Path to the Delft-FEWS REGION_HOME directory.
```

Comment:
Is REGION_HOME an environment variable that we can use as default here?

Reply:
It is a variable that can be used inside FEWS configurations, but I don't think it is commonly set as an environment variable. @gijsber can confirm.